assets/lang/ja.ini

```diff
@@ -596,7 +596,7 @@ tstr_AuxUIQuickStartOverlaysBody=Desktop+ を使用すると、デスクトッ
 tstr_AuxUIQuickStartOverlaysBody2=[+] を押して、リストからキャプチャ ソース/オーバーレイタイプを選択すると、新しいオーバーレイを作成できます。\nWeb ブラウザ オーバーレイなどの一部のタイプは、それぞれのアプリケーション コンポーネントがインストールされている場合にのみ使用できます。\n\n個々のオーバーレイまたは完全なレイアウトをプロファイルに保存できます。\n現在のオーバーレイ設定は、セッション間で自動的に記憶されます。
 tstr_AuxUIQuickStartOverlayPropertiesHeader=オーバーレイプロパティ
 tstr_AuxUIQuickStartOverlayPropertiesBody=オーバーレイにはさまざまなカスタマイズオプションがあります。\n原点と表示モードは、オーバーレイが表示される場所とタイミングを設定します。\nオーバーレイが表示されるはずなのに見つからない場合は、これらのプロパティを確認してください。\n前者が役に立たない場合は、その位置をリセットすると役立つこともあります。\n\nオーバーレイをモーションコントローラーの近くにドラッグして、モーションコントローラーにドッキングすることもできます。
-tstr_AuxUIQuickStartOverlayPropertiesBody2=一部のプロパティはデフォルトで非表示になっています。\nすべてのオプションを表示するには、設定ウィンドウで「詳細設定を表示」を切り替えます。\n\nUIウィンドウの位置は、それぞれのボタンを長押しすることでリセットできます。\nダブルクリックや右クリックは、クイックショートカットトグルと同様にオーバーレイボタンでも機能します。
+tstr_AuxUIQuickStartOverlayPropertiesBody2=一部のプロパティはデフォルトで非表示になっています。\nすべてのオプションを表示するには、設定ウィンドウで[詳細設定を表示]を切り替えます。\n\nUIウィンドウの位置は、それぞれのボタンを長押しすることでリセットできます。\nダブルクリックや右クリックは、クイックショートカットトグルと同様にオーバーレイボタンでも機能します。
```

Comment:
This seems to do the opposite of what I meant in the other comment. 「 」as quotation marks are fine and good when there originally were double quotes ("). The words originally enclosed by plain square brackets in the English text are what should stay as-is, though, as they're not meant to be quotation marks but illustrate the text being a button label.

At the end of the day this is a stylistic choice, however. So it's not a big deal overall and I just wanted to clarify. 
It's also within scope of changes I can safely do myself after the merging so I'm not going to be more annoying about it than I already am.

Reply:
I will use square brackets to unify them in this translation.